bench/Cargo.toml

```diff
@@ -1,17 +1,21 @@
 [package]
-name = "db-bench"
+name = "stress"
```

Comment:
A bit of git screwup. This should be reviewed as a new file, not a rename. `bench/core/` does not exist anymore.